saleor/graphql/core/utils/__init__.py

```diff
@@ -36,6 +36,11 @@ def str_to_enum(name):
 
 def validate_image_file(file, field_name):
     """Validate if the file is an image."""
+    print(type(file))
+    if not file:
+        raise ValidationError(
+            {field_name: ValidationError("File is required", code="invalid")}
```

Comment:
Not sure what's the best way to use error codes as enums here so we can keep it for now, but the error code should be `required`.